main.tf

```diff
@@ -15,33 +17,37 @@ resource "aws_route53_record" "autodiscover" {
   records = [local.autodiscover_record]
 }
 
-// SES identity / verification
-resource "aws_ses_domain_identity" "identity" {
+
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ses_domain_identity
+resource "aws_ses_domain_identity" "main" {
   domain = local.zone_name
 }
 
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record
 resource "aws_route53_record" "verification_token" {
   zone_id = var.zone_id
   name    = "_amazonses.${local.zone_name}"
   type    = "TXT"
   ttl     = "600"
-  records = [aws_ses_domain_identity.identity.verification_token]
+  records = [aws_ses_domain_identity.main.verification_token]
 }
 
-// DKIM
-resource "aws_ses_domain_dkim" "dkim" {
-  domain = aws_ses_domain_identity.identity.domain
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ses_domain_dkim
+resource "aws_ses_domain_dkim" "main" {
+  domain = aws_ses_domain_identity.main.domain
 }
 
+# see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record
 resource "aws_route53_record" "dkim" {
   count   = 3
   zone_id = var.zone_id
-  name    = "${element(aws_ses_domain_dkim.dkim.dkim_tokens, count.index)}._domainkey.${local.zone_name}"
+  name    = "${element(aws_ses_domain_dkim.main.dkim_tokens, count.index)}._domainkey.${local.zone_name}"
```

Comment:
You could index directly instead of using `element` which is a TF 0.11 type syntax.

Suggested:
```
records = ["${aws_ses_domain_dkim.main.dkim_tokens[count.index]}.dkim.amazonses.com."]
```

Reply:
Nope - removed and ignored!